participants/anthony.json

```
@@ -0,0 +1,36 @@
+{
+    // required
+    "name": "Anthony Craig",
+    // optional
+    "company": "your company name",
```

```suggestion
    "company": "your company name",
```

Can you update your company name? If you don't want to mention one, feel free to just remove this property.

---

I do not want to mention one.